tools/shell/tests/test_autocomplete.py

```diff
@@ -0,0 +1,332 @@
+# fmt: off
```

Comment:
Can we set a lower line-length for this file instead of completely disabling black?

Reply:
I really don't want to, personally Black has been more of an annoyance than a helpful tool

I value formatting and my style clashes immensely with that of Black

Reply:
I have no strong opinions on the formatting of Python code but I think there's a lot of value in not having to bikeshed code formatting and having it be automatically fixed by a formatter. If you don't like the formatting of Black perhaps we can switch to a different formatter instead of selectively disabling it in certain Python files?